src/maths/Vector3D.cpp

```
@@ -99,6 +99,18 @@ Math::Vector3D &Math::Vector3D::normalize() {
+  double len = this->length();
+  if (len != 0) {
+    return Vector3D(x / len, y / len, z / len);
+  }
+  return *this;
+}
+
 Math::Vector3D Math::cross(const Math::Vector3D &A, const Math::Vector3D &B) {
   return {A.y * B.z - A.z * B.y, A.z * B.x - A.x * B.z, A.x * B.y - A.y * B.x};
 }
+
+Math::Vector3D Math::Vector3D::reflect(const Math::Vector3D& incident, const Math::Vector3D& normal) {
```

Cppcheck (2.10-2)

[style] The function 'reflect' is never used.

(unusedFunction)